mappings/net/minecraft/item/LinkedItemStackHashSet.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/unmapped/C_voakmxnc net/minecraft/item/LinkedItemStackHashSet
+	FIELD f_bbmejudb strategy Lit/unimi/dsi/fastutil/Hash$Strategy;
```

Comment:
```suggestion
	FIELD f_bbmejudb STRATEGY Lit/unimi/dsi/fastutil/Hash$Strategy;
```

Reply:
hm, i swear that it was a private final field